File: fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/error/InvalidSignatureException.java
package com.github.dedis.popstellar.utility.error;

import com.github.dedis.popstellar.model.network.method.message.data.Data;
import com.github.dedis.popstellar.model.objects.security.Signature;

public class InvalidSignatureException extends InvalidDataException {

  public InvalidSignatureException(Data data, Signature signature) {
    super(data, "signature", signature.getEncoded());
  }
}

File: fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/error/InvalidSignatureException.kt
package com.github.dedis.popstellar.utility.error

import com.github.dedis.popstellar.model.network.method.message.data.Data
import com.github.dedis.popstellar.model.objects.security.Signature

class InvalidSignatureException(data: Data, signature: Signature) :
    InvalidDataException(data, "signature", signature.encoded)
